### PR TITLE
Miro c2c

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/miro-source.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+If the Source is configured with the **SIEM forward** option, the metadata field `_siemparser` will be set.
+
+| Field Name | Value |
+| --- | --- |
+| _siemparser | #TODO |
+
+## Prerequisite
+
+To get audit logs, you must have Miro Enterprise account.
```
Comment (stacykor) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/miro-source.md:26]: To get audit logs, you must have a Miro Enterprise account.

Comment (stacykor) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/miro-source.md:26]: that a is missing.

**Review by stacykor — CHANGES_REQUESTED** [review]
Just the one small change.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/miro-source.md (hunk 1 above)
Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/miro-source.md]: @stacykor Yeah that's a miss. Thank you for pointing that out.

**Review by kimsauce — APPROVED** [review]

**Review by stacykor — APPROVED** [review]

**Pull request merged**